### PR TITLE
All event screen styling

### PR DESCRIPTION
**Pull request by jaeheekim051510**
I added styling to all event screen!

@dbaile6 @janelleany @pmattam

### CODE REVIEW

File: src/Components/Header.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Link } from "react-router-dom";
 
 import mainLogo from '../images/foodoo-logo.png';
-import filterIcon from '../images/filter-icon.svg';
+import filterIcon from '../images/filter-Icon.svg';
```
Comment (janelleany) [inline src/Components/Header.js]: failed to compile. filename for the icon is filter-icon.

**Comment by janelleany** [comment]
it looks _really_ good! this is what I see when I change that one typo:

screen shot 2018-05-02 at 4 11 38 pm

bitmoji

**Comment by jaeheekim051510** [comment]
screen shot 2018-05-02 at 4 25 49 pm

**Comment by jaeheekim051510** [comment]
I had an error with icon and I see capital I in my folders.

**Pull request merged**